background_estimation/predMacros/makeBETrees.C

```diff
@@ -176,22 +180,44 @@ public:
             lep1PT_   = selectedLepton->pt();
             lep1ETA_  = selectedLepton->eta();
 
-            hwwChi_  = hwwChi;
-            hwwLi_  = hwwLi;
-            wlnuMass_ = wlnu.mass();
-            wlnuPT_    = wlnu.pt();
-
-            wjjTau2o1_ = wjjCand->tau2otau1();
-            wjjMass_   = wjjCand->sdMom().mass();
-            wjjPT_     = wqq.pt();
+            if(wjjCand) {
+                hwwChi_  = hwwChi;
+                hwwLi_  = hwwLi;
+                hwwPT_ = hWW.pt();
+                wlnuMass_ = wlnu.mass();
+                wlnuPT_    = wlnu.pt();
+                wjjTau2o1_ = wjjCand->tau2otau1();
+                wjjMass_   = wjjCand->sdMom().mass();
+                wjjPT_     = wqq.pt();
+            } else {
+                hwwChi_  = 0;
+                hwwLi_  = 0;
+                hwwPT_ = 0;
+                wlnuMass_ = 0;
+                wlnuPT_    = 0;
+                wjjTau2o1_ = 0;
+                wjjMass_   = 0;
+                wjjPT_     = 0;
+            }
 
-            hbbMass_ = hbbMass;
-            hbbPT_ = hbbCand->pt();
-            hbbCSVCat_ = hbbCSVCat;
-            hwwPT_ = hWW.pt();
-            hhMass_  = hh.mass();
-            hhMassBasic_   = hh_basic.mass();
-            nAK4Btags_   = std::min(nMedBTags_HbbV,250);
+            if(hbbCand) {
+                hbbMass_ = hbbMass;
+                hbbPT_ = hbbCand->pt();
+                hbbCSVCat_ = hbbCSVCat;
+                nAK4Btags_   = std::min(nMedBTags_HbbV,250);
```

Comment:
This should be filled in all cases. If there is no hbb the nMedBTags_HbbV will include all b tags